scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -864,3 +864,86 @@ task SummarizeTaskMonitorLogs {
     File monitoring_summary = "monitoring_summary.txt"
   }
 }
+
+# Note - this task should probably live in GvsCreateFilterSet, but I moved it here when I was refactoring VQSR Classic out of
+# GvsCreateFilterSet (in order to avoid a circular dependency)
+# When VQSR Classic is removed, consider putting this task back in GvsCreateFilterSet
```

Comment:
Maybe worth adding as a comment to https://broadworkbench.atlassian.net/browse/VS-992?

Reply:
So commented.